src/codeu/chat/client/ClientUser.java

```diff
@@ -88,8 +91,8 @@ public void showCurrent() {
     printUser(current);
   }
 
-  public void addUser(String name) {
-    final boolean validInputs = isValidName(name);
+  public void addUser(String name, String password) {
```

Comment:
I think it would need the password argument since the password is now a part of the User entity.